website/server/controllers/top-level/payments/stripe.js

```diff
@@ -124,9 +131,25 @@ api.subscribeEdit = {
   middlewares: [authWithHeaders()],
   async handler (req, res) {
     let token = req.body.id;
+    let groupId = req.body.groupId;
```

Comment:
hmm actually I see there's no validation for any of the params for these routes even before this PR